qutebrowser/config/configdata.yml

```diff
@@ -1829,11 +1819,21 @@ messages.unfocused:
 
 ## prompt
 
-prompt.filebrowser:
```

Comment:
You'll need to keep the original key with a `renamed: ...`, see e.g. the changes in d2e69454ce18a8cc972eb39f0ec457eb66155fdc for a similar change.

You will also adjust the code using `prompt.filebrowser` to use the new name.

Reply:
Would I just do this and nothing else? Or would I keep the original parameters.
```yml
prompt.filebrowser:
    renamed: prompt.filebrowser.enabled
```

Reply:
Yep, just that. This will then tell qutebrowser to e.g. adjust existing `autoconfig.yml` files (when someone configured this via `:set`) accordingly.

Reply:
Nice. I was wondering how this was going to work because otherwise it could conflict.